tabcmd/locales/en/extra.properties

```diff
@@ -0,0 +1,22 @@
+# We pull in the whole file for tabcmd_messages_en.properties and shared_wg_en.properties,
+# but just a couple of useful strings from the other half dozen files
+
+# rest_api_en.properties
+bad_request.detail.invalid_content_type=“{0}” is not a valid content type.
```

Comment:
Are the curly quotes intentional? Seems odd for a console app.

Reply:
They were just copied direct from product strings but you're right, seems out of place. I added a step to replace them all with '